core/templates/pages/subtopic-viewer-page/navbar-pre-logo-action/subtopic-viewer-navbar-pre-logo-action.component.ts

```diff
@@ -17,30 +17,46 @@
  *  of the subtopic viewer.
  */
 
-require('domain/classroom/classroom-domain.constants.ajs.ts');
+
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { ClassroomDomainConstants } from
+  'domain/classroom/classroom-domain.constants.ts';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { UrlService } from 'services/contextual/url.service';
 require('domain/utilities/url-interpolation.service.ts');
 require('services/contextual/url.service.ts');
```

Comment:
Remove these `requires`, we don't need them anymore. A general rule of thumb, angular components and services use only `imports`.

Reply:
Sorry, this was a mistake. Just slipped through my mind removing these.